packages/styles/src/theme.ts

```diff
@@ -227,6 +243,22 @@ export const darkPalette = {
     paper: colors.cinder,
     paperSecondary: colors.darkBlueMagenta,
     paperLight: colors.jaguar,
+    paperExtraLight: colors.cinder,
+    amountField: colors.charadeLight,
+    containedButton: colors.blackRussian,
+    warning: colors.blackRussian,
+    card: colors.steelGray,
+    productCard: colors.darkSpace,
```

Comment:
I think we should avoid such specific names in the akroweb theme. Same for the "poolAddress". Can we make it more general?